it/assets/BaseIntegrationTestConstants.scala

```diff
@@ -29,4 +29,6 @@ object BaseIntegrationTestConstants {
   val testYear = "2018"
   val testCalcType = "it"
 
+  val testSelfEmpId = "aSelfEmpId"
```

Comment:
Can we call this `testIncomeSourceId` and give it a style that is more consistent with a realistic value. e.g. "XAIT000000123456564"